CHANGELOG.md

```diff
@@ -4,10 +4,25 @@ All notable changes to [Earthly](https://github.com/earthly/earthly) will be doc
 
 ## Unreleased
 
+## v0.8.5 - 2024-03-11
+
+### Added
+
+- Added `--aws` flag to `RUN` command which makes AWS environment variables or ~/.aws available. Enable with the `VERSION --run-with-aws` feature flag. [#3803](https://github.com/earthly/earthly/issues/3803)
+- Added `--allow-privileged` flag to `FROM DOCKERFILE` command. Enable with the `VERSION --allow-privileged-from-dockerfile` feature flag. Thanks to [@dustyhorizon](https://github.com/dustyhorizon) for the contribution! [#3706](https://github.com/earthly/earthly/issues/3706)
+
 ### Fixed
 
 - Fixes an issue where wildcard `BUILD`'s are invoked from a relative directory (e.g., an `Earthfile` containing `BUILD ./*+test` invoked with `earthly ./rel-dir+target`). [#3840](https://github.com/earthly/earthly/issues/3840)
 - `--pass-args` will no longer pass builtin args, which would result in `value cannot be specified for built-in build arge errors. [#3775](https://github.com/earthly/earthly/issues/3775)
+- Fixes an issue where `BUILD --auto-skip` is invoked with wildcard builds. [#3862](https://github.com/earthly/earthly/issues/3862)
```

Comment:
```suggestion
- Fixes a parsing issue with `BUILD` flag arguments and wildcard targets [#3862](https://github.com/earthly/earthly/issues/3862)
```

Reply:
did you intentionally remove `--auto-skip`?

Reply:
Yes, the issue wasn't related to `--auto-skip` but rather any flag used with wildcards.

Reply:
@mikejholly might be a good idea to quickly update the referenced ticket, in case someone follows this link

Reply:
Sure. I updated the title. The original report assumed the issue had to do with `--auto-skip`, but the referenced PR describes the actual problem.